src/Tyr-GPU-Driver.md

```diff
@@ -88,9 +83,8 @@ can check our [issue
 board](https://gitlab.freedesktop.org/panfrost/linux/-/issues/?label_name%5B%5D=tyr).
 We will be posting good starting tasks at a future point.
 
-To work on any given task, assign it to yourself and follow up with a merge
-request against `tyr-for-upstream` (for patches targeting the upstream driver)
-or `tyr-dev` (for work in the downstream staging branch).
+To work on any given task, assign it to yourself. For patches targeting the downstream
+staging branch, open a merge request against `tyr-dev`.
```

Comment:
I would suggest mentioning the upstream route too or removing both, instead of mentioning only the downstream case (the text before the change has both too).

(But please see my comment above)